yaki_admin/src/stores/roleStore.ts

```diff
@@ -10,6 +10,8 @@ export const useRoleStore = defineStore("roleStore", {
     customersIdWhereAmCaptain: [] as number[],
     // all captains id a user can have
     captainsId: [] as number[],
+    customerId: 0 as number,
+    captainId: 0 as number,
```

Comment:
what is the use of thoses attributes ?

Reply:
i deleted captainId: 0 as number because i didn't need here. but I need to have customerId et array of captainsId for refresh after the deletion.